### PR TITLE
[CHANGE] Added instance.Feature.PageNavigation that can be used to enable/disable all the ways that navigate the viewer to a different page

### PR DESCRIPTION
**Pull request by ZhijieZhang**
This is an attempt to address #401 and https://trello.com/c/pMfbUglx/645-new-api-to-override-mouse-wheel. This PR doesn't explicitly expose an API to disable mouse wheel, but I think in many cases if the customer wants to disable wheel then he probably wants to disable other ways that can change the page number.

To verify:
In console, do instance.disableFeatures([instance.Feature.PageNavigation]), and verify that the page number can't be changed by using wheel, the swiping gesture, arrow up and arrow down.

### CODE REVIEW

File: src/helpers/createFeatureAPI.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import core from 'core';
 import hotkeys from 'src/apis/hotkeys';
 import localStorageManager from 'helpers/localStorageManager';
-import TouchEventManager from 'helpers/TouchEventManager';
+import touchEventManager from 'helpers/TouchEventManager';
```
Comment (achenPDFTron) [inline src/helpers/createFeatureAPI.js]: seeing `hotkeysManager` is camel cased, should we do the same with `TouchEventManager` to make it consistent?

**Comment by achenPDFTron** [comment]
@ZhijieZhang when im in non-mobile mode, pressing page - up and page -down allows me to transition between page?

File: src/helpers/createFeatureAPI.js [hunk 2]
```diff
@@ -129,6 +130,27 @@ export default (enable, store) => features => {
         store.dispatch(actions.setThumbnailMultiselect(enable));
       },
     },
+    [Feature.PageNavigation]: {
+      fn: () => {
+        const { up: arrowUpHandler, down: arrowDownHandler } = hotkeysManager.keyHandlerMap;
```
Comment (achenPDFTron) [inline src/helpers/createFeatureAPI.js]: i think we need to add in page up and page down as well? as i can still navigate the page?

**Comment by achenPDFTron** [comment]
@ZhijieZhang thats all my comments for now

File: src/constants/feature.js [hunk 3]
```diff
@@ -14,6 +14,7 @@
  * @property {string} Copy Ability to copy text or annotations use Ctrl/Cmd + C hotkeys or the copy button.
  * @property {string} ThumbnailMerging Ability to drag and drop a file into the thumbnail panel to merge
  * @property {string} ThumbnailReordering Ability to reorder pages using the thumbnail panel
+ * @property {string} PageNavigation Ability to navigate through pages using mouse wheel, arrow up/down keys and the swipe gesture.
```
Comment (achenPDFTron) [inline src/constants/feature.js]: im guessing we left out page up / page down on purpose?

**Comment by ZhijieZhang** [comment]
> seeing `hotkeysManager` is camel cased, should we do the same with `TouchEventManager` to make it consistent? 

Hmm `TouchEventManager` is also camel cased. I guess you mean the first character should be lowercased? yeah I think we should make the naming consistent. I've added a card to my trello board and will do it in another PR.

**Comment by ZhijieZhang** [comment]
> i think we need to add in page up and page down as well? as i can still navigate the page? 

ohhh good cacth, I missed these two keys. I will add them, thanks!

**Pull request merged**